sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -113,6 +158,15 @@ _current AS (
     COALESCE(first_session.adjust_creative, metrics.adjust_creative) AS adjust_creative,
     COALESCE(first_session.adjust_network, metrics.adjust_network) AS adjust_network,
     metrics.install_source AS install_source,
+    metrics.latest_adjust_campaign AS latest_adjust_campaign,
```

Comment:
Not 100% sure about this, curious for opinions. My reasoning is that we don't need to coalesce here as in `init.sql` because if this table has already been initialized, we only want to overwrite the latest value if we've received a new metrics or baseline ping as opposed to for some reason seeing a later first session ping or `first_seen` entry

Reply:
This is probably fine 99.9% of the time. I would expect a few edge cases where we're missing a latest-reported entry b/c it came from a late-arriving first-session ping.